packages/slate-react/src/components/android/diff-text.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * Returns the number of characters that are the same at the beginning of the
+ * String.
+ *
+ * @param {String} prev  the previous text
+ * @param {String} next  the next text
+ * @returns {Number | null} the offset of the start of the difference; null if there is no difference
```

Comment:
Maybe remove old pre-typescript commenting style. Only mentioned once but should apply to all methods.

Reply:
When I attempted to remove the types from the documentation, the eslint started failing:

>   1:1  error  Missing JSDoc return type                   valid-jsdoc
>   5:4  error  Missing JSDoc parameter type for 'prev'     valid-jsdoc
>   6:4  error  Missing JSDoc parameter type for 'next'     valid-jsdoc

It appears that the parameter type is still required, even though it is redundant in typescript.

Reply:
That's unexpected since there are other files in the repo that don't have them and they don't raise eslint errors. I wonder if it's an all or nothing deal. If there are some JSDoc comments, then the entire file has to be? It does appear that jsdoc does appear in `.eslintrc`. I feel like we can just remove that from the `.eslintrc`. I just did a search through the Slate repo and it is used in exactly one method call which probably was left there unintentially `packages/slate-react/src/utils/lines.ts`

Reply:
Removed eslint rule and removed jsdoc comments

Reply:
Removed eslint rule and removed jsdoc comments